### ISSUE TITLE
WCAG 3.1.1: Ensures every HTML document has a lang attribute (html)

### ISSUE THREAD

**Issue by niccokunzmann** [niccokunzmann/open-web-calendar#347]
#### Issue
`Ensures every HTML document has a lang attribute` ([html-has-lang](https://accessibilityinsights.io/info-examples/web/html-has-lang))
#### Target application
[Open Web Calendar](https://open-web-calendar.hosted.quelltext.eu/calendar.html?)
#### Element path
html
#### Snippet
`<html>`
#### How to fix
    Fix any of the following:
      The <html> element does not have a lang attribute
#### Environment
Chrome version 123.0.0.0
This accessibility issue was found using Accessibility Insights for Web 2.41.0 (axe-core 4.7.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).

<small>
<hr/>
We're using <a href="https://polar.sh/niccokunzmann">Polar.sh</a> so you can upvote and help fund this issue. We receive the funding once the issue is completed & confirmed by you. Thank you in advance for helping prioritize & fund our work.
</small>

<a href="https://polar.sh/niccokunzmann/open-web-calendar/issues/347">
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://polar.sh/api/github/niccokunzmann/open-web-calendar/issues/347/pledge.svg?darkmode=1">
  <img alt="Fund with Polar" src="https://polar.sh/api/github/niccokunzmann/open-web-calendar/issues/347/pledge.svg">
</picture>
</a>

**Comment by joshhazelhurst123** [comment]
Hello, I can only see 2 html pages that need updating to include the below. <html lang="{{ specification['language'] }}">

- one-day-event.html
- about.html

I can see the page below already has the tag. Therefore, I believe that a pull request needs to be done to update those 2 html pages. 

index.html --> <html lang="{{ specification['language'] }}">

**Comment by niccokunzmann** [comment]
That is good news! The about.html is the only important one and the other one is only for tests.

**Comment by niccokunzmann** [comment]
https://github.com/niccokunzmann/open-web-calendar/blob/master/templates/calendars/dhtmlx.html would need it too.

**Comment by niccokunzmann** [comment]
#361 fixes the about.html file.

**Comment by niccokunzmann** [comment]
Thanks! I will close this issue now... I can re-open a new one if I see that there are problems arising :)
Would you like to add it to the changelog? If not, I will do that for the next release.

**Comment by joshhazelhurst123** [comment]
Great I have updated the changelog.

**Comment by joshhazelhurst123** [comment]
ah yes, here it is. Thought I had already made the pull request for the changelog. See here for the new pull request. Update changelog.md #373
https://github.com/niccokunzmann/open-web-calendar/pull/373
